stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,88 @@
+- if the letter is not in the word, **one body part** should be added to the gallows and the incorrect guesses counter should be updated accordingly;
+- the clicked/pressed letter should be disabled and it should be reflected in the UI *(for example, by using a different color on the disabled button)*;
+6. The body parts appear on the gallows in the following order: head, body, left arm, right arm, left leg, right leg. It means that the user has *6 attempts* to guess the word.
+7. The player will continue guessing letters until either the word is solved or all six body parts are on the gallows.
+8. At the end of the game, a modal window should appear. It should:
+- include the message that either congratulates or informs the user that the game has been lost;
+- the secret word (independently whether it was guessed correctly or not);
+- a "play again" button.
+9. When the user clicks on "play again", the game should start over:
+- the gallows should be empty;
+- the question should be changed;
+- the blank lines should be updated according to the new secret word.
+- an incorrect guesses counter should be set to 0.
+
+### Approximate appearance
+*This example is purely illustrative. Feel free to change the layout as you wish.*
+![screenshot](hangman.png)
+
+### Helpful materials
+At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangman-game?type=design&node-id=0%3A1&mode=design&t=4Gj7Ayo0yckppNo8-1) you will find:
+- examples of the game flow,
+- body parts (for those who don't want to design them by themselves).
+
+### Main functional requirements
+- The design should be adaptive (or responsive), that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything should be displayed correctly, all functionality should be present, nothing should disappear or leave beyond the screen.
+- Initially, body in the index.html file must be empty, all necessary elements are generated using JS.
+- At the beginning of the game, the question-answer pair should be chosen randomly.
+- The application can be done in English or in Russian language.
+
+## Repository requirements
+- The task should be done in private school's repository.
+- The code should be committed to a separate branch named as the task.
+- The `main` (or old style master) branch should be empty (contain only files like README.md or .gitignore).
+- Commit messages should follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name should contain **the task name**.
+- The pull request description should be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
```

Comment:
_The same suggestion to use the word "must" instead of "should"_

**Suggested:** - The task must be done in private school's repository.
- The code must be committed to a separate branch named as the task.
- The `main` (or old style master) branch must be empty (contain only files like README.md or .gitignore).
- Commit messages must follow the [guideline](https://docs.rs.school/#/git-convention)
- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
- The pull request name must contain **the task name**.
- The pull request description must be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).

Reply:
Done